index.d.ts

```diff
@@ -1,3 +1,3 @@
 declare module 'is-url-http' {
-  export default function isUrlHttp(url: string): boolean;
-}
\ No newline at end of file
+  export = function isUrlHttp(url: string): boolean;
```

Comment:
For weird syntax reasons, it’s not quite inlinable like that
```suggestion
  function isUrlHttp(url: string): boolean;
  export = isUrlHttp;
```